app/scripts/modules/core/src/help/help.contents.ts

```diff
@@ -420,6 +420,8 @@ const helpContents: { [key: string]: string } = {
   'pipeline.config.webhook.terminalStatuses':
     'Comma-separated list of strings that will be considered as TERMINAL status.',
   'pipeline.config.webhook.customHeaders': 'Key-value pairs to be sent as additional headers to the service.',
+  'pipeline.config.webhook.failFastCodes':
+    'Comma separated HTTP status codes (4xx or 5xx) that will cause this webhook stage to fail without retying.',
```

Comment:
* should probably be `Comma-separated` just to be consistent with the other help text on the stage right above this entry - we are not consistent throughout Deck on "comma separated" vs. "comma-separated".
* s/retying/retrying

Reply:
👍